cirq-core/cirq/ion/ion_device.py

```diff
@@ -28,6 +28,17 @@ class IonDevice(devices.Device):
     Qubits have all-to-all connectivity.
     """
 
+    gateset = ops.Gateset(
```

Comment:
Nit: can we avoid declaring this as a class level attribute (style guide discouraged) and move it into the `__init__` ?

Reply:
sure, will do.

Reply:
Added a module level helper to create the gateset and moved `gateset` to `__init__`.  LMK if this looks good.